src/styles/main.scss

```diff
@@ -1,3 +1,218 @@
-body {
+$blue: #00acdc;
+$dark-blue: #060b35;
+$gray: #616070;
+$white: #fff;
+$font-family: "Roboto", sans-serif;
+
+* {
   margin: 0;
+  font-family: $font-family;
+}
+
+html {
+  color: $dark-blue;
+  font-size: 8px;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+
+  padding: 0 50px;
+
+  background: #fff;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+}
```

Comment:
create separated file for each BEM block, using BEM allow you to use nesting in scss - a big power that it gives